CredentialProvider.Microsoft/Resources.resx

```diff
@@ -339,7 +339,11 @@ Cache Location
 Windows Integrated Authentication
     {13}
         Boolean to enable/disable using silent Windows Integrated Authentication
-        to authenticate as the logged-in user. Enabled by default.</value>
+        to authenticate as the logged-in user. Enabled by default.
+
+Device Flow Authentication Timeout
+    {14}
+        Device Flow authentication timeout in seconds. Default is 90 seconds.</value>
```

Comment:
Please also update readme.md (and yes, it's annoying that it isn't automatically generated yet).